pkg/oc/bootstrap/clusterup/components/registry/registry_install.go

```diff
@@ -55,13 +62,24 @@ func (r *RegistryComponentOptions) Install(dockerClient dockerhelper.Interface,
 	imageRunHelper := run.NewRunHelper(dockerhelper.NewHelper(dockerClient)).New()
 	glog.Infof("Running %q", r.Name())
 
-	securityClient, err := securityclient.NewForConfig(r.ClusterAdminKubeConfig)
-	if err != nil {
-		return err
-	}
-	err = openshift.AddSCCToServiceAccount(securityClient, "privileged", "registry", "default", os.Stdout)
+	err = retry.RetryOnConflict(retry.DefaultRetry, func() error {
+		securityClient, err := securityclient.NewForConfig(restConfig)
+		if err != nil {
+			return err
+		}
+		privilegedSCC, err := securityClient.SecurityContextConstraints().Get("privileged", metav1.GetOptions{})
+		if err != nil {
+			return err
+		}
+		privilegedSCC.Users = append(privilegedSCC.Users, serviceaccount.MakeUsername("default", "registry"))
```

Comment:
openshift.DefaultNamespace ?

Reply:
> openshift.DefaultNamespace ?

Depending on constants makes for unpleasant depedencies.  That one in particular doesn't seem good.